env.yaml

```diff
@@ -0,0 +1,18 @@
+name: macrophage-striation
+channels:
+  - bioconda
+  - conda-forge
+  - anaconda
+  - defaults
+dependencies:
+  - python>=3.10
+  - pip==22.3.1
+  - numpy==1.23.5
+  - pandas==1.5.2
+  - pillow==9.2.0
+  - pyqt==5.15.7
+  - ipykernel==6.14.0
+  - jupyter==1.0.0
+  - pip:
+      - black==22.6.0
+      - pre-commit==2.20.0
```

Comment:
Hey @d33bs , thanks for pointing this out. I do have a small question about this. Since I use pre commit hooks:
 - [isort](https://github.com/PyCQA/isort)
 - [black](https://github.com/psf/black) 
 - [sourcery](https://github.com/sourcery-ai/sourcery)

Should I leave the `.pre-commit-config.yaml` in the remote repo? This is my first time using git pre-commit hooks :)

Reply:
I would recommend including it. Including that file helps make sure it stays with the repository and can enable collaborators to use the same pre-commit configuration (unifying code style or quality expectations). Also, if you're including `black` as part of your `pre-commit` checks, you might consider removing it from the conda environment.